contents/docs/advanced/proxy.mdx

```diff
@@ -20,7 +20,7 @@ You then use this subdomain as your `api_host` in the initialization of PostHog
 
 Options for deploying a reverse proxy include:
 
-- [PostHog's own Managed Reverse Proxy](/docs/advanced/proxy/managed-reverse-proxy) (Included with the Teams addon)
+- [PostHog's managed reverse proxy](/docs/advanced/proxy/managed-reverse-proxy) (Recommended)
```

Comment:
I'm curious to see if we get any feedback here, I think people have been loudly annoyed (on Twitter) when features are gated behind the teams plan. Saying upfront is a way to prevent this, but 🤷